src/pages/_app.js

```diff
@@ -1,13 +1,56 @@
 import App, { Container } from 'next/app'
 import React from 'react'
 
+import Cookie from 'js-cookie'
+
+// Temporary fix to fetch dependency on 'fetches' lib
+if (typeof window === 'undefined') {
+  const fetch = require('isomorphic-fetch')
+}
+
+import { createClient, getHTTPMethods } from 'fetches'
+import { Provider } from 'react-fetches'
+
 class ProjectApp extends App {
+  static getInitialProps(context) {
+    const { ctx } = context
+    const getTokenFromLocalCookie = () => Cookie.get('token')
```

Comment:
To make this `_app.js` more clear and straightforward, maybe in the future we can create some util functions to handle these cookies (or just send this functions to a util file), what you think? The idea behind it is to make this app file more easily understood by anyone.

Reply:
You say to create a function called getToken and send just the Cookie.get('token') right??
and the string 'token' could be a env var called REACT_APP_TOKEN_TEXT, what do you think?